hermes-console/src/components/tracking-card/TrackingCard.spec.ts

```diff
@@ -27,14 +27,14 @@ describe('TrackingCard', () => {
     const elements = container.querySelectorAll('a')!!;
     expect(elements[0]).toHaveAttribute('href', 'https://test-tracking-url1');
     expect(elements[0]).toHaveTextContent('url1');
-    expect(elements[1]).toHaveAttribute('href', 'https://test-tracking-url2');
-    expect(elements[1]).toHaveTextContent('url2');
+    expect(elements[2]).toHaveAttribute('href', 'https://test-tracking-url2');
```

Comment:
what happened with element 1?

Reply:
There are only two urls, I think the index 1 url contained something different, but i don;t remember what, i'll check and come back with the answer

Reply:
have you possibly checked that?